manifests/base/runtimes/data_cache/rbac.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cache-initializer-sa
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cache-initializer-role
+rules:
+  - apiGroups: ["*"]
+    resources:
+      - leaderworkersets
+      - services
+      - serviceaccounts
+    verbs: ["create", "get", "list", "watch"]
```

Comment:
you might need to grant access to delete service account: https://github.com/kubeflow/trainer/blob/master/pkg/initializers/dataset/cache.py#L310